securesignals-ima-devapp/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.dev'
+    compileSdk 33
```

Comment:
Nit: Not required as defined in `common.gradle`

Reply:
Done.